src/java/org/apache/cassandra/db/compaction/unified/Controller.java

```diff
@@ -53,6 +55,14 @@ public class Controller
     private final static String DEFAULT_SCALING_PARAMETERS =
         CassandraRelevantProperties.UCS_SCALING_PARAMETER.getString();
 
+    /**
+     * The minimum sstable size. Sharded writers split sstables over shard only if they are at least as large as the
+     * minimum size.
+     */
+    static final String MIN_SSTABLE_SIZE_OPTION = "min_sstable_size";
```

Comment:
We currently have special code to change the base shard count to 1 for system tables (see `DEFAULT_BASE_SHARD_COUNT` below) because they are expected to be small. The min size makes this unnecessary; can we remove that special case as part of the commit that changes the default min size?